http3/server.go

```diff
@@ -210,16 +210,14 @@ func (s *Server) ListenAndServe() error {
 //
 // If s.Addr is blank, ":https" is used.
 func (s *Server) ListenAndServeTLS(certFile, keyFile string) error {
-	var err error
-	certs := make([]tls.Certificate, 1)
-	certs[0], err = tls.LoadX509KeyPair(certFile, keyFile)
+	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
```

Comment:
changed to not allocate the slice with a make before we are sure we don't have any errors 
and if all ok no need for the make we can directly add to the tls.Config struct

Reply:
I don't really care about allocs in the error case.